README.md

```diff
@@ -16,8 +16,14 @@ With skaffold
 make test
 ```
 
-With docker-compose
+With docker-compose or locally
 
 ```
-SUBSTRAT_CONFIG_FILE=values-docker-compose.yaml make test
+SUBSTRAT_CONFIG_FILE=local-values.yaml make test
```

Comment:
There is a PR that adresses this `values-docker-compose.yaml` file specifically : https://github.com/SubstraFoundation/substra-tests/pull/5/files

I think the renaming of this file (if one is necessary) should move to that PR instead of this one.

Reply:
This PR https://github.com/SubstraFoundation/substra-tests/pull/5/files has been created after this one.
I don't really know what to do in this case...

Do you think I should still use the `values-docker-compose.yaml` filename even if it does not only apply to docker?

Reply:
I was nitpicking here, sorry, I shouldn't have. Let's keep the name as it is, if someone feels the need to rename it later, it'll be their issue.